deploy/crds/iam.policies_v1alpha1_iampolicy.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We can't update the CRD version by just changing the apiversion. We need to regenerate them with their API.

Reply:
For these files the automation generation will not work because they are manually created and copied here, I have tried to make the change by looking at the other files, manually made it conform to v1 definition, please check and review the same.